pages/work/enigma.vue

```diff
@@ -0,0 +1,219 @@
+          platform, employers can review their responses and overall performance
+          and then use these findings to better evaluate the candidate’s
+          application.
+        </p>
+      </div>
+
+      <div
+        class="
+          bg-enigma-blue
+          flex flex-wrap
+          h-64
+          md:h-140
+          justify-center
+          items-center
+        "
+      >
+        <div class="bg-enigma-blue w-1/2 lg:w-1/3">
+          <inline-svg
+            class="fill-white h-auto w-full"
+            src="/svgs/clients/enigma.svg"
+          />
+        </div>
+      </div>
+
+      <div class="p-4 lg:p-12">
+        <h2>Challenge</h2>
+
+        <p>
+          The Enigma assessment application was in a very raw proof of concept
+          state when Ship Shape was brought in to act as the development team
+          that would create a production-ready product. Enigma already had
+          customers signed up and ready to start utilizing the application for
+          their recruiting needs, but still had a substantial roadmap of
+          features in need of implementation. Ship Shape’s challenge was to
+          implement essential features, including the full assessment workflow
+          and corresponding administrative tasks, in order to launch Enigma. We
+          continued to help build out the application based on customers’
+          feedback about their additional needs.
+        </p>
+      </div>
+
+      <div class="bg-grey-light flex justify-center lg:flex-1">
+        <div class="p-4 lg:p-12">
+          <nuxt-img
+            alt="Enigma Login"
+            class="flex"
+            format="webp"
+            src="/img/work/enigma/enigma-login.png"
+          />
+        </div>
+      </div>
+
+      <div class="p-4 lg:p-12">
+        <h2>Solution</h2>
+
+        <h3>Next.js</h3>
+
+        <p>
+          The development team made use of
+          <a href="/https://nextjs.org/">Next.js</a> to prioritize quick loading
+          times on the client side of this application. The server-side
+          rendering capabilities of Next.js allows the Enigma application to
+          retrieve web pages and any corresponding data in one go, eliminating
+          the need for multiple network requests and preventing subsequent
+          delays in data population.
+        </p>
+
+        <h3>Shepherd Tour</h3>
+
+        <p>
+          The Ship Shape team also had the opportunity to integrate one of our
+          own open-source projects,
+          <a href="https://shepherdjs.dev/docs/">Shepherd</a>, into the Enigma
+          platform. The Shepherd library provides a way to guide users through a
+          tour of a web application, highlighting the key features users should
+          familiarize themselves with in order to use the app at its full
+          potential.
+        </p>
+      </div>
+
+      <div class="bg-grey-light flex justify-center lg:flex-1">
+        <div class="p-4 lg:p-12">
```

Comment:
```suggestion
        <div class="pl-4 pr-4 pt-4 lg:pl-24 lg:pr-24 lg:pt-12">
```

I think to match the format of the other case studies, image containers should have these classes?

Reply:
Oops, you're right. I accidentally looked at other case studies' SVG containers as a reference. Reverting now.